js/test/date/testdatefmt_es_CO.js

```diff
@@ -26,1191 +26,6 @@ if (typeof(GregorianDate) === "undefined") {
 if (typeof(DateFmt) === "undefined") {
     var DateFmt = require("../../lib/DateFmt.js");
 }
-function tCOSTDateFmtConstructorEmpty_es_CO() {
```

Comment:
These ones are not legacy -- they are just in the wrong section. They need to be moved to within the module.exports.testdatefmt_es_CO, then renamed "tCOST" -> "test", then given test.expect() and test.done() calls and convert asserts to test.asserts, then converted to comma-separated functions. These tests do not exist below.